index.html

```diff
@@ -571,70 +571,98 @@ <h3>External Binary Content</h3>
+        <p>
+        <ul>
+          <li><code>copy</code> - requests that the server dereference the external content URI and treat that as if
+            it were the entity body of the request.</li>
+          <li><code>redirect</code> - requests that the server record the location of the external content and
+            handle requests for that content using HTTP redirect responses with the <code>Content-Location</code>
+            header specifying the external content location. See also <a href="#redirect-and-proxy"></a>.</li>
+          <li><code>proxy</code> - requests that the server  record the location of the external content and
+            handle requests for that content by proxying. See also <a href="#redirect-and-proxy"></a>.</li>
+        </ul>
+        <p>
+          Fedora servers MUST reject with a 4xx range status code requests for which the <code>handling</code>
+          attribute is not present or cannot be respected.
         </p>
-        <p id='external-content-content-location'>
-          <code>GET</code> and <code>HEAD</code> responses for any external <a>LDP-NR</a> SHOULD include a
-          <code>Content-Location</code> header with a URI representation of the location of the external content if
-          the Fedora server is proxying the content.
+        <p id='external-content-media-type'>
+          Fedora servers MUST use the value of the <code>type</code> attribute in the external content link as
+          the media type of the external content, if provided. Otherwise:
         </p>
-        <p id='external-content-want-digest'>
-          <code>GET</code> and <code>HEAD</code> requests to any external <a>LDP-NR</a> MUST correctly
-          respond to the <code>Want-Digest</code> header defined in [[!RFC3230]].
+        <ul>
+          <li>Servers MAY use the media type specified in a <code>Content-Type</code> header of the request.
+          <li>Servers MAY use the media type obtained when accessing the external content via the specified
+            scheme (e.g. the <code>Content-Type</code> header for external content accessed via
+            <code>http</code>).</li>
+          <li>Servers MAY use a default media type.</li>
+          <li>Servers MAY reject the request with a 4xx range status code.</li>
+        </ul>
+        <p id='external-content-body-response-headers'>
+          A Fedora server receiving requests that would create or update an <a>LDP-NR</a> with content external to
+          the request entity MUST reject request if it cannot guarantee all of the response headers required by the
+          <a>LDP-NR</a> interaction model in this specification.
         </p>
-        <section id='external-content-caveats'>
-          <h4>Referenced RDF Content in Mandatory LDP Serializations</h4>
+        <section id='external-content-options'>
+          <h4>Advertizing External Content Support</h4>
```

Comment:
Should be "Advertising".  http://grammarist.com/spelling/advertise-vs-advertize/

Reply:
Indeed, I've got too used to switching all my s to z!